### PR TITLE
Finish #4308: Documentation for the new PropertyNormalizer

### PR DESCRIPTION
**Pull request by wouterj**
Finishes #4308 

| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#9708) |
| Applies to | 2.6+ |
| Fixed tickets | #4272 |

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -238,6 +244,28 @@ When serializing, you can set a callback to format a specific object property::
...
+    This normalizer reads the content of the class by calling the "getters"
+    (public methods starting with "get"). It will denormalize data by calling
+    the constructor and the "setters" (public methods starting with "set").
+
+    Objects are serialized to a map of property names (method name stripped of
+    the "get" prefix and converted to lower case) to property values.
+
+:class:`Symfony\\Component\\Serializer\\Normalizer\\PropertyNormalizer`
+    This normalizer directly reads and writes public properties as well as
+    **private and protected** properties. Objects are serialized to a map of
```
Comment (cordoval) [inline components/serializer.rst]: are normalized instead of serialized?

File: components/serializer.rst [hunk 2]
```diff
@@ -50,6 +50,12 @@ which Encoders and Normalizer are going to be available::
 
     $serializer = new Serializer($normalizers, $encoders);
 
+There are several normalizers available, e.g. the
+:class:`Symfony\\Component\\Serializer\\Normalizer\\GetSetMethodNormalizer` or the
+:class:`Symfony\\Component\\Serializer\\Normalizer\\PropertyNormalizer`.
+To read more about them, refer to the `Normalizers`_ section in this page.
```
Comment (xabbuh) [inline components/serializer.rst]: of this page

File: components/serializer.rst [hunk 3]
```diff
@@ -50,6 +50,12 @@ which Encoders and Normalizer are going to be available::
...
+All the examples shown below use the GetSetMethodNormalizer.
```
Comment (xabbuh) [inline components/serializer.rst]: We should enclose `GetSetMethodNormalizer` with literals. It's a class name.

**Comment by weaverryan** [comment]
This reads really nicely Wouter - thanks for finishing this off :)

**Pull request merged**